### ISSUE TITLE
fn2 functionality "bugs" ?

### ISSUE THREAD

**Issue by matosimi** [matosimi/atari-fontmaker#6]
I have found comments for moderation on my web... which were invisible until now...  2 bugreports regarding fn2 functionality:

![image](https://github.com/user-attachments/assets/de183bf5-17be-4264-84aa-b595307d15e9)

It has been a long time since I used fn2 ... I assume i abandoned it at some time and only kept it for the backwards compatibility... so "Load 1" button can load the double font... I tried it and "RES S" is actually working ok ...
I used doublefont or dualfont only in one project that has never saw the light of the world
[font_dual.zip](https://github.com/user-attachments/files/16830554/font_dual.zip)

I'm not sure how to handle these bugreports, but I think the backwards compatibility is good enough... and I simply recommend to not use fn2 anymore, but save it as 2 separate fonts :)

@cycoPH what do you think?

**Comment by CycoPH** [comment]
I’ll take a look and see what I find.  Will report back asap.

- I fixed the first issue with the case of .fn2 giving wrong results.
- I would not add the save .fn2 functionality. Rather use the view save.  Much better and it lets you save all your test screens.
- The RES S buttons issue was solved by loading a .fn2 file and then auto saving the data in two different files:
- hello.fn2 would then be split into "hello-fn2-1.fnt" and "hello-fn2-2.fnt", similarly for the second font bank.
- That would mean pressing RES S will just load the data from the split files
